wikidict/lang/it/__init__.py

```diff
@@ -48,7 +50,10 @@
 templates_italic: Dict[str, str] = {}
 
 # Templates more complex to manage.
-templates_multi: Dict[str, str] = {}
+templates_multi: Dict[str, str] = {
+    # {{Term|statistica|it}} -> <small>(<i>statistica</i>)</small>
```

Comment:
No need to be so specific ;)
```suggestion
    # {{Term|statistica|it}}
```